core-utils/src/main/java/com/anytypeio/anytype/core_utils/OnSwipeListener.kt

```diff
@@ -34,8 +34,8 @@ abstract class OnSwipeListener : SimpleOnGestureListener() {
         //  C => it's a DOWN swipe
         //  D => it's a LEFT swipe
         //
-        val x1 = e1.x
-        val y1 = e1.y
+        val x1 = e1?.x ?: 0f
```

Comment:
please if UX behavior related to this code is not broken

Reply:
I checked everything works, but the view this listener is working with will be removed very soon